### ISSUE TITLE
Keeps keyboard focus after alt tabbing - Bug returned

### ISSUE THREAD

**Issue by Juts2** [Kyusung4698/PoE-Overlay#627]
# 🐞 bug report

**Describe the bug**
Alt tab and I cannot hit ctrl+T while in chrome/firefox because the overlay has keyboard focus. Same bug that was fixed previously is back. On latest version from 4-13. 

**To Reproduce**

**Expected behavior**

**Screenshots**

## 🌍  Your Environment

- OS:  e.g. Win7 x64
- Version:  e.g. 0.6.9 portable
- PoE:  e.g. Steam 3.10.0/e English

**Comment by Kyusung4698** [comment]
Hi, thanks for creating an issue. This is intended while any dialog is open the overlay will not remove the hotkeys. Either close all dialogs or use the internal browser.

**Comment by MissingL-tter** [comment]
I don't think this behaviour is intuitive and is perhaps what is causing issues with apps like iCue.

As a user, I expect that if I do any action other than interact with an overlay window, the overlay would close. That no longer seems to be the case with PoE Overlay. `0.6.20` behaves much more intuitively in my opinion but hey, maybe I'm an outlier.

**Comment by widedev** [comment]
Can I unbind Ctrl+T for poe-overlay? It's sooo annoying... I have to exit poe-overlay everytime I want to browse in Firefox.

"Either close all dialogs or use the internal browser."
-> In my case there are no dialogs and I don't want to use the internal browser for private browsing. The only way to prevent poe-overlay from distracting my browser seems to be to kill the application. Why not give the user the choice to remove key-bindings?

**Comment by Juts2** [comment]
> Can I unbind Ctrl+T for poe-overlay? It's sooo annoying... I have to exit poe-overlay everytime I want to browse in Firefox.
> 
> "Either close all dialogs or use the internal browser."
> -> In my case there are no dialogs and I don't want to use the internal browser for private browsing. The only way to prevent poe-overlay from distracting my browser seems to be to kill the application. Why not give the user the choice to remove key-bindings?

This is my experience as well. No dialog open and it still intermittently intercepts ctrl t

**Comment by widedev** [comment]
Everytime when I die I have to exit poe-overlay, active my auras (including Ctrl+T) and restart poe-overlay. Please overthink this...

**Comment by Kyusung4698** [comment]
## 0.6.23 (2020-04-18)

- fix shortcut still active even though poe overlay has no more focus (#627)

**Comment by Bazeloth** [comment]
I can confirm this is fixed in 0.6.23
